### PR TITLE
Add schedule for 2024 courses

### PR DESCRIPTION
**Pull request by adamltyson**
@neuroinformatics-unit/neuroinformatics-core 

I've raised this PR to add the information for the 2024 courses we're giving. I decided to make a single course listing the schedule for all the material we're giving (creating 8 courses seemed a bit much). 

Could you all flesh out the website with more details on each course as you have it (content, materials etc). I guess this can be either:
- Details on the main schedule page
- A link to an existing/new course page

It would be good to get this fleshed out soon so we can start advertising.

### CODE REVIEW

**Comment by niksirbi** [comment]
Just to clarify, are you fine with us repurposing the older specific course pages (in cases where the same course as last year is being taught), updating the schedule there, and then linking that page in the general one you've created here?

**Comment by adamltyson** [comment]
Fine by me :)

File: docs/source/courses/software-skills.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+- 14:00-17:00: Linux and high-performance computing (SWC Ground Floor Lecture Theatre)
+
+### Monday, October 7th
+- 11:00-12:00: General microscopy: basics (SWC Brasserie Seminar Room)
+- 13:00-14:30: General microscopy: applications (SWC Brasserie Seminar Room)
+- 14:30-18:00: Histology analysis (1) (SWC Brasserie Seminar Room)
+
+### Tuesday, October 8th
+- 10:00-11:00: Histology analysis (2) (SWC Ground Floor Lecture Theatre)
+- 14:00-17:00: Histology analysis (3)  (SWC Brasserie Seminar Room)
```
Comment (niksirbi) [inline docs/source/courses/software-skills.md:35]: I personally think the schedule would be much more readable as a table.
```suggestion
| Date                  | Time           | Event                                                        | Location                           |
|-----------------------|----------------|--------------------------------------------------------------|------------------------------------|
| **Monday, September 30th** | 13:00-14:00    | Introduction to the course                                    | SWC Brasserie Seminar Room         |
|                       | 14:00-17:00    | Introduction to Python (1)                                    | SWC Brasserie Seminar Room         |
| **Tuesday, October 1st**   | 10:00-11:00    | Data management and sharing (1)                                | SWC Ground Floor Lecture Theatre   |
|                       | 15:00-17:00    | Data management and sharing (2)                                | SWC Ground Floor Lecture Theatre   |
| **Wednesday, October 2nd** | 10:00-12:30    | Introduction to Python (2)                                    | GCNU Seminar Room                  |
|                       | 13:30-17:00    | Version control and software development best practices       | GCNU Seminar Room                  |
| **Thursday, October 3rd**  | 10:00-13:00    | Video-based analysis of animal behaviour (1)                  | SWC Brasserie Seminar Room         |
|                       | 14:00-17:00    | Video-based analysis of animal behaviour (2)                  | SWC Brasserie Seminar Room         |
| **Friday, October 4th**    | 14:00-17:00    | Linux and high-performance computing                          | SWC Ground Floor Lecture Theatre   |
| **Monday, October 7th**    | 11:00-12:00    | General microscopy: basics                                    | SWC Brasserie Seminar Room         |
|                       | 13:00-14:30    | General microscopy: applications                              | SWC Brasserie Seminar Room         |
|                       | 14:30-18:00    | Histology analysis (1)                                        | SWC Brasserie Seminar Room         |
| **Tuesday, October 8th**   | 10:00-11:00    | Histology analysis (2)                                        | SWC Ground Floor Lecture Theatre   |
|                       | 14:00-17:00    | Histology analysis (3)                                        | SWC Brasserie Seminar Room         |
```

Comment (adamltyson) [reply docs/source/courses/software-skills.md]: Great, thanks!

**Comment by adamltyson** [comment]
I'm going to merge this so we can start advertising, feel free to raise PRs with any changes.

**Comment by niksirbi** [comment]
FYI @adamltyson the short description for the HPC course is still missing in software-skills.md

**Comment by adamltyson** [comment]
Yeah I just noticed that. I'll add it now.

**Pull request merged**